js/utils/constants.js

```diff
@@ -585,6 +591,34 @@
       VOLUME_CHANGED: 'volumeChanged',
 
       /**
+       * A request to change the mute state has been made.
+       * The handler is called with the following arguments:
+       * <ul>
+       *   <li>The desired mute state of the video element.</li>
+       *   <li>The id of the video on which to mute (as defined by the module that controls it).
+       *        If null or undefined, all video elements volume will be changed</li>
+       * </ul>
+       *
+       *
+       * @event OO.EVENTS#CHANGE_MUTE_STATE
+       */
+      CHANGE_MUTE_STATE: 'changeMuteState',
+
+      /**
+       * The mute state has changed.
+       * The handler is called with the following arguments:
+       * <ul>
+       *   <li>The current mute state of the video element.</li>
+       *   <li>The id of the video on which to mute (as defined by the module that controls it).
```

Comment:
Just a minor nitpick, should this be in past tense to reflect that this already happened?

Reply:
Yep, I'll update this.